homeassistant/components/logbook.py

```diff
@@ -301,6 +308,10 @@ def _get_events(hass, config, start_day, end_day):
                     & (Events.time_fired < end_day)) \
             .filter((States.last_updated == States.last_changed)
                     | (States.state_id.is_(None)))
+
+        if entity_id is not None:
+            query = query.filter(States.entity_id == entity_id.lower())
```

Comment:
`entity_id` is always lower case, enforced by `ha-entity-picker` on frontend

Reply:
I think it's good to lowercase it as it's a general API.